integration-tests/internal/misc-hostfns/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "misc_hostfns"
+description = "E2E tests for various host functions"
+version = "6.0.0-alpha.1"
+authors = ["Use Ink <ink@use.ink>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+ink = { path = "../../../crates/ink", default-features = false, features = ["unstable-hostfn"] }
+
+[dev-dependencies]
+ink_e2e = { path = "../../../crates/e2e", features = ["sandbox"] }
```

```suggestion
ink_e2e = { path = "../../../crates/e2e" }
```

Tests don't appear to need the `sandbox` feature